### PR TITLE
nginx migration to 1.1.1

### PR DESCRIPTION
**Pull request by dominiquemongelli**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Nginx ingress controller migration to version 1.1.1

https://deploy-preview-144--streams-open-docs.netlify.app/docs/install/
https://deploy-preview-144--streams-open-docs.netlify.app/docs/relnotes/20220228_streams_relnotes/
https://deploy-preview-144--streams-open-docs.netlify.app/docs/architecture/#nginx

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

File: content/en/docs/relnotes/20220228_streams_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+### Streams third-parties
+
+The following third-party library has been upgraded:
+
+* Nginx Ingress Controller : `1.1.1`
+
+### Kubernetes version
+
+Due to the migration to nginx in version `1.1.1`, the minimum version of kubernetes supported is `1.1.19`.
```
Comment (andreamussap) [inline content/en/docs/relnotes/20220228_streams_relnotes.md:25]:
```suggestion
As a result of the Nginx library update to version `1.1.1`, the minimum supported version of Kubernetes is now `1.1.19`.
```

**Comment by cremond** [comment]
Looks good to me from a technical standpoint. Ready for your review @andreamussap

File: content/en/docs/relnotes/20220228_streams_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+linkTitle: Streams February 2022 Release Notes
+weight: 144
+date: 2021-01-27
+---
+
+## Summary
+
+Streams is available as a set of Docker containers deployable in Kubernetes by using a Helm chart. For a summary of the system requirements, see [Install Streams](/docs/install/).
+
+## Important changes
```
Comment (andreamussap) [inline content/en/docs/relnotes/20220228_streams_relnotes.md:12]: Don't you want to inform about the release of Salesforce Publisher beta feature?

**Review by rbissey-axway — APPROVED** [review]
Looks good to me.

**Review by cremond — APPROVED** [review]
The "downtime" documentation was short lived, but it's because we fixed it :) Looks good to me.

File: content/en/docs/install/_index.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ export REGISTRY_SERVER="repository.axway.com"
 kubectl create secret docker-registry "${REGISTRY_SECRET_NAME}" --docker-server="${REGISTRY_SERVER}"  --docker-username="${REGISTRY_USERNAME}" --docker-password="${REGISTRY_PASSWORD}" -n "${NAMESPACE}"
 ```
 
-To use your Kubernetes Secret in the registry, add the Secret's name in the `imagePullSecrets` array.
+To use your Kubernetes Secret in the registry, add the Secret's name in the `imagePullSecrets` array. For example, add `--set imagePullSecrets[0].name="${REGISTRY_SECRET_NAME}"` to the Helm chart installation command.
```
Comment (guillaumelamirand) [inline content/en/docs/install/_index.md:97]: The precision about how to customize imagePullSecrets shouldn't be added here and is explained in a dedicated section: https://develop--streams-open-docs.netlify.app/docs/install/customize-install/#options-to-manage-helm-parameters

**Review by guillaumelamirand — CHANGES_REQUESTED** [review]
See comment about --set

**Review by guillaumelamirand — APPROVED** [review]

**Pull request merged**